blueprints/addon/files/README.md

```diff
@@ -1,26 +1,47 @@
-# <%= addonName %>
 
```

Comment:
Can this blank line go away?

Reply:
It can, I usually use it as padding, but I'm okay if we decided to remove it

Reply:
I vote for removal, it looks strange to me.

Reply:
removed